.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+os:
+ - linux
+ - osx
```

Comment:
Really OSX? We don't want to test vibe.d, and aren't using any platform specific stuff.